### ISSUE TITLE
Autocomplete doesn't have arrow keyboard navigation

### ISSUE THREAD

**Issue by HanzSchi** [HaizeaAnalytics/climate_stats#1]
Autocomplete option field on home page doesn't support choosing with arrow keys.
